evaluator/src/main/kotlin/OrtResultRule.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2021 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.evaluator
+
+import org.ossreviewtoolkit.model.Identifier
+import org.ossreviewtoolkit.model.OrtResult
+
+/**
+ * A [Rule] to check an [OrtResult].
+ */
+open class OrtResultRule(
+    ruleSet: RuleSet,
+    name: String,
+
+    /**
+     * The [OrtResult] to check.
+     */
+    val ortResult: OrtResult,
+) : Rule(ruleSet, name) {
+    override val description = "Evaluating ORT result rule '$name'."
+
+    override fun issueSource() = "$name - ORT result"
+
+    init {
+        matchers += RuleMatcher.MATCH_ALWAYS
```

Comment:
This should not be required, if no matchers are configured the default behavior is already that the rule always applies.

Reply:
Oh, thanks. I must have misunderstood the code then.